packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -292,5 +289,6 @@ DataViewsSubComponents.LayoutSwitcher = ViewTypeMenu;
 DataViewsSubComponents.Pagination = DataViewsPagination;
 DataViewsSubComponents.Search = DataViewsSearch;
 DataViewsSubComponents.ViewConfig = DataviewsViewConfigDropdown;
+DataViewsSubComponents.Footer = DataViewsFooter;
```

Comment:
I've exported Footer as a subcomponent, as [Jay suggested](https://github.com/WordPress/gutenberg/pull/71173#issuecomment-3209640416). I don't think I have a calibrated instinct to say if this is a good idea or not. It's better than having to use the classes, agree to that. Though I'm not sure at which point we stop making DataViews components public API. All things considered, I'm fine with going ahead with this.